platform/darwin/src/MGLSymbolStyleLayer.h

```diff
@@ -619,7 +619,8 @@ MGL_EXPORT
 
 /**
  Value to use for a text label. Feature properties are specified using tokens
- like {field_name}.
+ like {field_name}.  (Token replacement is only supported for literal text-field
+ values--not for data-driven property functions.)
```

Comment:
Ah, okay.  I suppose, in that case, that the least confusion-causing thing to do might be to just _drop_ the parenthetical in the `style-spec-overrides-v8.json`, since there's no established vocabulary for these function types yet?

@1ec5 what do you think?